init.lua

```diff
@@ -38,6 +38,13 @@ local table = {
     remove  = table.remove
 }
 local helpers = require("vicious.helpers")
+local dstatus, debug = pcall(require, "gears.debug")
+local warn
+if dstatus then
+  warn = debug.print_warning
+else
+  warn = function (msg) io.stderr:write('Warning (vicious): ', msg, '\n') end
```

Comment:
io.stderr should be localized here, also for the consistency with the few lines above, double quote might be preferred.